install_config/registry/extended_registry_configuration.adoc

```diff
@@ -80,42 +80,58 @@ We recommend against rebooting the entire cluster because that incurs
 unnecessary downtime for pods and does not actually clear the cache.
 
 [[whitelisting-docker-registries]]
-== Whitelisting Docker Registries
-
-You can specify a whitelist of docker registries, allowing you to curate a set
-of images and templates that are available for download by {product-title}
-users. This curated set can be placed in one or more docker registries, and then
-added to the whitelist. When using a whitelist, only the specified registries
-are accessible within {product-title}, and all other registries are denied
-access by default.
+== Configuring an External Registry Search List
+
+You can use the *_/etc/containers/registries.conf_* file to create a list of 
+Docker registries to search for container images. 
+
+The *_/etc/containers/registries.conf_* file is a list of registry servers 
+that {product-title} should search against when a user pulls an image using
+the image short name. For example, if the registry search list contains 
+`registry.redhat.io`, `docker.io`, and `registry.example.com`, and the 
+image short name is `myimage:latest`, {product-title} searches 
+the registries in that order until it finds the `myimage:latest`. 
+
+The *_/etc/containers/registries.conf_* file allows you to customize the 
+order of the search, specify secure and insecure registries, and define a blocked
+registry list. 
+
+{product-title} can pull images from any secure registry, which is a registry 
+that uses SSL/TLS, or insecure registries, which require standard web protocols 
```

Comment:
I'd be surprised if _standard web protocols_ isn't just HTTP. So I'd say an insecure registry just uses HTTP.